benchmark-tools/mysql/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.8"
+services:
+  db:
+    image: mysql:5.7
+    environment:
+      - MYSQL_ALLOW_EMPTY_PASSWORD="no"
```

Comment:
This is confusing. Should this be `- MYSQL_ALLOW_EMPTY_PASSWORD="yes"`, or something that makes it clear that we actually are allowing an empty password and we're going to use it to connect to the database?